File: w10/d02/Nelson_Schubart/mta_js/spec/mta_spec.js
var jas = require('jasmine-node');

var mta_object = require('../src/mta.js');

var MTA = mta_object.MTA;

// describe ("MTA", function () {
//   it("should say hello", function() {
//     expect(MTA.hello()).toEqual("Greetings!")
//   });
// });

var $n_line = {"Times Square": 0, "34th St": 1, "28th St N-line": 2, "23rd St N": 3, "Union Square": 4; "8th St N": 5};

var $l_line = {"8th Ave": 0, "6th Ave": 1, "Union Square": 2, "3rd Ave": 3, "Union Square": 4, "1st Ave": 5};

var $six_line = {"Grand Central": 0, "28th on the 6-line": 1, "23rd on the 6-line": 2, "Union Square" 3, "Astor Place": 4};

function chompStop() {
  console.log("What stop are you on?");
      var currentStop = gets.chomp;
      console.log("You entered: #{currentStop}";
  };



var array_keys = new Array();
var array_values = new Array();

for (var key in a) {
    array_keys.push(key);
    array_values.push(a[key]);
}

alert(array_keys);
alert(array_values);

File: w10/d02/Christopher_Bajorin/hw/spec/mta_spec.rb
require 'mta'

RSpec.describe Subway do

let(:trip) { Subway.new }
let(:n_line) { ["Times Square", "34th", "28th-N", "23rd", "Union Square",  "8th"] }
let(:l_line) { ["8th", "6th", "Union Square", "3rd", "1st"] }
let(:six_line) { ["Grand Central", "33rd", "28th-6", "23rd", "Union Square", "Astor Place"] }

  it "should tell the user what stop they enter" do
    expect(trip.enter("Times Square", n_line)).to eq("You're at Times Square")
  end

  it "should tell the user what stop they exit" do
    expect(trip.exit("8th", l_line)).to eq("You will exit at 8th")
  end

  # it "should notify the user of the start index" do
  #   trip.enter("34th", n_line)
  #   trip.exit("8th", l_line)
  #   expect(trip.start()).to eq(1)
  # end

  # it "should notify the user of the stop index" do
  #   trip.enter("Times Square", n_line)
  #   trip.exit("8th", l_line)
  #   expect(trip.end()).to eq(0)
  # end

  it "should notify the user how many stops" do
    trip.enter("Times Square", n_line)
    trip.exit("8th", l_line)
    expect(trip.length()).to eq(6)
  end

end
